distribution/lib/Standard/Searcher/0.1.0/src/Data_Science.enso

```diff
@@ -0,0 +1,71 @@
+## Enso is, first and foremost, a tool for putting you in direct contact with
```

Comment:
Needs more marketing. I'd start with "Enso is, first and foremost, a way of life". Also make sure to mention `Inventing on Principle` by Bret Victor.